app/experimenter/experiments/tests/api/v5/test_queries.py

```diff
@@ -1251,6 +1251,37 @@ def test_branches(self):
                 experiment_data["treatmentBranches"],
             )
 
+    @parameterized.expand([[True], [False]])
```

Comment:
Funny I just had this conversation with @yashika-khurana the other day, I tend to use parameterized for cases where different states will trigger different code paths.  Here we're just always spitting out verbatim the value, it's declared as a `boolean` field type which will enforce values and so on, so I wouldn't consider it necessary to parametrically test both values.